src/Containers/BooksList.js

```diff
@@ -1,19 +1,30 @@
+/* eslint-disable consistent-return */
 /* eslint-disable no-unused-vars */
```

Comment:
With the ESLint inline comment, you are indirectly changing the linter's rules by disabling these rules, let's fix the codebase instead.


If you are using VSCode, you can delete the inline comment and hover over the error(configure Eslint checks locally). And you will see your options. Click on the more info button that will take you to a GitHub repo where it is explained why you are getting this error. I find this feature very educational. Please, try to learn and implement the necessary rules so you don't have to keep this disabling line on top of each file. I believe you can do this! 💪😇